### PR TITLE
CAMEL-12478: Allow use of custom keyboard

### PR DESCRIPTION
**Pull request by willianantunes**
As described [here](https://issues.apache.org/jira/browse/CAMEL-12478), it will permit the usage of custom keyboard in order to increase the interaction with the user as it's a lot easier to select an option instead of ask the user writing one.

### CODE REVIEW

**Comment by willianantunes** [comment]
I separated in 5 commits, but when the review is done I should probably squash them up in one, right?

File: components/camel-telegram/src/main/docs/telegram-component.adoc [hunk 1]
```diff
@@ -310,3 +310,60 @@ to("telegram:bots/123456789:AAE_dLq5C19xwGjw3yiC2NvEUrZcejK21-Q987654321:AAE_dLq
...
 
 Note that the corresponding URI parameter is simply `chatId`.
+
+### Customizing keyboard
+
+You can customize the user keyboard instead of asking him to write an option. `OutgoingTextMessage` has the property `ReplyKeyboardMarkup` which can be used for such thing.
+
+[source,java]
+---------------------------------------------------------
+from("telegram:bots/123456789:AAE_dLq5C19xwGjw3yiC2NvEUrZcejK21-Q987654321:AAE_dLq5C19xwOmg5yiC2NvSrkT3wj5Q1-L")
```
Comment (WillemJiang) [inline components/camel-telegram/src/main/docs/telegram-component.adoc]: Please make sure you don't use a private bots, as the access key may be abused.

Comment (willianantunes) [reply components/camel-telegram/src/main/docs/telegram-component.adoc]: I used the same access key available in the documentation. See other examples on it. This key is not mine.

Comment (nicolaferraro) [reply components/camel-telegram/src/main/docs/telegram-component.adoc]: So it should be a random one I've generated (I hope :smile:)

**Comment by oscerd** [comment]
LGTM now, @WillemJiang would you like to merge?

**Comment by oscerd** [comment]
Thanks, this has been merged on master.

**Pull request closed without merging**